supervisor/addons/validate.py

```diff
@@ -282,8 +283,9 @@ def _migrate(config: Dict[str, Any]):
         vol.Optional(ATTR_SERVICES): [vol.Match(RE_SERVICE)],
         vol.Optional(ATTR_DISCOVERY): [valid_discovery_service],
         vol.Optional(ATTR_SNAPSHOT_EXCLUDE): [str],
-        vol.Optional(ATTR_SNAPSHOT_PRE): str,
-        vol.Optional(ATTR_SNAPSHOT_POST): str,
+        vol.Optional(ATTR_BACKUP_EXCLUDE): [str],
```

Comment:
Need to be done with the migration function and need test

Reply:
Not doing any more with renaming until _everyone_ is on the same page on what we should/shouldn't rename